modules/vpn/main.tf

```diff
@@ -107,13 +107,13 @@ resource "aws_iam_access_key" "furyagent" {
 }
 
 resource "aws_iam_policy_attachment" "furyagent" {
-  name       = "${var.name}-vpn"
+  name       = "${var.name}-${var.vpn_bucket_name_prefix}"
   users      = [aws_iam_user.furyagent.name]
   policy_arn = aws_iam_policy.furyagent.arn
 }
 
 resource "aws_iam_policy" "furyagent" {
-  name = "${var.name}-${var.vpc_id}-${data.aws_region.current.name}-vpn"
+  name = "${var.name}-${var.vpc_id}-${data.aws_region.current.name}-${local.vpn_bucket_iam_name_prefix}"
```

Comment:
I would leave vpn here, and deal with it in the migration procedure if necessary.